db/seeds/fincap_data/article_seeds.rb

```diff
@@ -20,108 +20,6 @@
-        of financial capability must be improved from their current low levels,
-        and that if everybody works together it is possible to rise to the
-        challenge. The changing financial environment makes this more important
-        than ever before.
-
-        ## **Why is it so important?**
-
-        $~youtube_video
-
-        JkhmfVGBkPM
-
-        The Financial Capability Strategy for the UK Why is it needed?
-
-        ~$
-
-        ## **What is in the Strategy?**
-
-        The Strategy sets out a clear description of the problem that needs to
-        be solved and why it matters. It describes what success looks like over
-        the lifetime of the Strategy and how the aims of the Strategy will be
-        achieved.
-
-        For the Strategy to be a success it is important that resources are
-        allocated to interventions that are proven to work.
-
-        The Strategy sets out the issues across a number of different themes
-        and for each of the devolved nations (Northern Ireland, Scotland and
-        Wales). We have produced a range of documents to allow stakeholders
-        to understand the different audiences:
-
-        1.  Executive Summary (4 pages)
-        2.  The Strategy (50 pages)
-        3.  The Full Strategy (150 pages)
-
-        The Full Strategy has been broken down into chapters for those with a
-        specific interest in any of the following areas:
-
-        *   Introduction and context
-        *   Evidence and evaluation
-        *   Children and young people
-        *   Young Adults
-        *   Working Age
-        *   Older People in Retirement
-        *   Retirement Planning
-        *   People in Financial Difficulties
-        *   Ease and accessibility
-        *   Northern Ireland
-        *   Scotland
-        *   Wales
-      CONTENT
-    ),
-    Comfy::Cms::Block.new(
-      identifier: 'component_hero_image',
-      content: '/assets/styleguide/hero-sample.jpg'
-    ),
-    Comfy::Cms::Block.new(
-      identifier: 'component_hero_description',
-      content: 'Financial capability across the UK'
-    ),
-    Comfy::Cms::Block.new(
-      identifier: 'component_cta_links',
-      content: <<-CONTENT
-        [Evidence Hub](/general_info)
-        [Evaluation Toolkit](/common-evaluation-toolkit)
-        [The Steering Groups](/steering-groups)
-        [2015 Financial Capability Survey](/financial-capability-survey)
-      CONTENT
-    ),
-    Comfy::Cms::Block.new(
-      identifier: 'component_download',
-      content: <<-CONTENT
-      [UK Strategy](/financial+capability+strategy.pdf)
-      [UK Detailed Strategy](/detailed-strategy.pdf)
-      [Key statistics on Financial Capability](/key-statistics.pdf)
-      [Financial Capability Progress Report 2017](/fincap+progress+report+2017.pdf)
-      CONTENT
-    ),
-    Comfy::Cms::Block.new(
-      identifier: 'component_feedback',
-      content: 'email@moneyadviceservice.org.uk'
-    )
-  ]
-)
-
```

Comment:
Will we not have an article page created on seeds then?

Reply:
@tomas-stefano that's a good point, maybe we can seed another article, but definitely with some other content (unrelated to UK Strategy). What do you think?

Reply:
Good point, maybe create the 'our-approach' article as it is required for one of the [homepage ctas](https://github.com/moneyadviceservice/fin_cap/blob/master/app/views/homepages/show.html.erb#L47).

Reply:
Is there any tests on fincap that relies on this data?

Reply:
If you mean 'our-approach' article, I think not

Reply:
I'm adding it then, it sound sensible to have one anyway. 👍

Reply:
@your I think it should be added as a separate pr, strictly speaking it should have been part of [this pr](https://github.com/moneyadviceservice/fin_cap/pull/174).

Reply:
@margOnline Fair enough, it would require [this test on fincap](https://github.com/moneyadviceservice/fin_cap/blob/master/features/article_page.feature) to be adapted anyway (after removing the cassette), so I would agree with you. Can do in a separate ticket/PR.

Reply:
@tomas-stefano other than the test for `our-approach` which I'm going to pick up, can this pr be approved?

Reply:
@your  I just realised that the fincap:setup task also needs to be amended to include the creation of the uk_strategy layout, refer to [pr #465](https://github.com/moneyadviceservice/cms/pull/465/files#diff-fc7b15bcd327a3dc679c3acba3682f47)

Reply:
@margOnline thanks for spotting it, added with d05a94b